packages/visx-responsive/src/components/ParentSize.tsx

```diff
@@ -1,12 +1,14 @@
 import debounce from 'lodash/debounce';
-import React from 'react';
+import React, { useEffect, useMemo, useRef, useState } from 'react';
 import ResizeObserver from 'resize-observer-polyfill';
 
 export type ParentSizeProps = {
   /** Optional `className` to add to the parent `div` wrapper used for size measurement. */
   className?: string;
   /** Child render updates upon resize are delayed until `debounceTime` milliseconds _after_ the last resize event is observed. */
   debounceTime?: number;
+  /** Optional keys provided won't trigger a state change when changed */
+  disableFor?: keyof ParentSizeState | (keyof ParentSizeState)[];
```

Comment:
Agree the name may not be ideal here, what about `ignoreDimensions`? `disable` doesn't quite seem right since we are still passing those dimensions, just ignoring their changes.

Reply:
Sure, ignoreDimensions works! Sounds better to me.